Functions.Templates/Templates/EventGridBlobTrigger-CSharp-Isolated/.template.config/template.json

```
@@ -0,0 +1,77 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "Blob",
+        "EventGrid"
+    ],
+    "name": "EventGridBlobTrigger",
+    "identity": "Azure.Function.CSharp.Isolated.EventGridBlobTrigger.3.x",
+    "groupIdentity": "Azure.Function.EventGridBlobTrigger",
+    "shortName": "eventgridblob",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "EventGridBlobTriggerCSharp",
+    "symbols": {
+        "namespace": {
+            "description": "namespace for the generated code",
+            "replaces": "Company.Function",
+            "type": "parameter"
+        },
+        "Connection": {
+            "description": "The name of the app setting containing your Storage connection.",
+            "type": "parameter",
+            "defaultValue": "",
+            "replaces": "ConnectionValue"
+        },
+        "Path": {
+            "description": "This is the path within your storage account that the trigger will monitor.",
+            "type": "parameter",
+            "defaultValue": "samples-workitems",
+            "replaces": "PathValue"
+        },
+        "HostIdentifier": {
+            "type": "bind",
+            "binding": "HostIdentifier"
+        }
+    },
+    "primaryOutputs": [
+        {
+            "path": "EventGridBlobTriggerCSharp.cs"
+        }
+    ],
+    "defaultName": "EventGridBlobTriggerCSharp",
+    "postActions": [
+        {
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs Nuget package",
+            "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+            "ContinueOnError": "true",
+            "ManualInstructions": [],
+            "args": {
+                "referenceType": "package",
+                "reference": "Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs",
+                "version": "6.3.0",
```

6.4.0 is out. Can use that one now.
```suggestion
                "version": "6.4.0",
```

---

Updated